rental/src/main/java/pl/agh/edu/rental/rental/RentalController.java

```diff
@@ -0,0 +1,48 @@
+package pl.agh.edu.rental.rental;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import pl.agh.edu.rental.auth.CurrentUser;
+import pl.agh.edu.rental.auth.User;
+import pl.agh.edu.rental.errors.NoRentalError;
+import pl.agh.edu.rental.errors.UserUnauthorizedError;
+import pl.agh.edu.rental.rental.dto.RentalCreateInput;
+import pl.agh.edu.rental.rental.dto.RentalData;
+import pl.agh.edu.rental.rental.service.RentalService;
+
+@CrossOrigin(origins = "http://localhost:3000")
```

Comment:
Does it mean, that the microservice will be on localhost:3000? Or it will only accept requests from this address? Either way, later we should maybe add some config files with these adresses in one place